setup.py

```diff
@@ -36,7 +36,8 @@ def find_version():
         'peewee>=3.9',
         'peewee-moves>=2.1.0,<3.0',
         'python-dotenv',
-        'webargs>=5.2.0,<6.0'
+        'webargs>=5.2.0,<6.0',
+        'PyJWT>=2.0.1'
```

Comment:
PyJWT shouldn't be a requirement for a Colibris project.

Reply:
It shouldn't but it is 😄 . The Colibiris code depends on it to work if configured to use authentication with JWT.

Reply:
Is there a way around it? Maybe an import with a dedicated exception message? It's far from ideal to have deps pulled in just in case.